src/app/vote/demo/layout.tsx

```diff
@@ -0,0 +1,30 @@
+'use client';
+import { useEffect, useState } from 'react';
+import { useCookies } from 'react-cookie';
+import { useRouter } from 'next/navigation';
+
+export default function DemoLayout({ children }: { children: React.ReactNode }) {
+  const router = useRouter();
+  const [cookies, setCookie, removeCookie] = useCookies();
```

Comment:
만약 hook의 구조분해할당을 통해 전달받은 함수중 사용하지 않을 함수나 변수의 경우는 _ 로 대채하면 추후 코드 이해에 좀 더 도움이 될수도 있을 것 같아요~!
```suggestion
  const [cookies, _ , _] = useCookies();
```